builds.yml

```diff
@@ -146,14 +146,28 @@ features:
       - ./{app,shared,ui}/**/institutional/**
   build-flask:
     assets:
-      - src: ./app/build-types/flask/images/
```

Comment:
This change seems unintentional?

Reply:
now I understand why I was seeing the MM logo instead of the Flask one! Thanks!

Reply:
reverted